src/web/mjs/connectors/MangaDex.mjs

```diff
@@ -94,11 +94,11 @@ export default class MangaDex extends Connector {
         uri.searchParams.append('contentRating[]', 'pornographic');
         const request = new Request(uri, this.requestOptions);
         const data = await this.fetchJSON(request, 3);
```

Comment:
You can deconstruct the result as follows `const {data} = await this.fetchJSON(request, 3);`, it's better, but it looks like it's working :)

Reply:
Thanks for the info! I didn't know about this. Then I'll rewrite as you suggested.